update-stemcell.sh

```diff
@@ -10,15 +10,14 @@
 # - docker daemon running on the host (requires volumes)
 
 set -o errexit -o nounset
-
-if [ "${USER}" != "jan" ]; then
-    REPO="${REPO:-../../github.com/SUSE/scf}"
-    HELM=helm
-else
-    REPO="${REPO:-../scf}"
+if [ "${USER}" == "jan" ]; then
     HELM=helm-2.11
```

Comment:
Ah, I read that condition backwards.

Could you get an `.envrc` that exports `$HELM`, and do `: "${HELM:=helm}"` here or something?  Basically, I'm trying to eliminate the `jan` check :)

Reply:
I won't do the `.envrc` because we seem to check them into git as well... 😦 

But I'm fine with invoking the script with `HELM=helm-2.11 ./update-stemcell.sh`, so will do the other part.